src/BenchmarkDotNet.Core/Reports/SummaryTableExtensions.cs

```diff
@@ -84,17 +84,13 @@ private static string BuildStandardText(SummaryTable table, string[] line, strin
             return buffer.ToString();
         }
 
-        private static string BuildBoldText(SummaryTable table, string[] line, string leftDel, string rightDel, int columnIndex)
+        private static string BuildBoldText(SummaryTable table, string[] line, string leftDel, string rightDel, int columnIndex, string boldMarkupFormat)
         {
-            const string markdownBold = "**";
-
             var buffer = GetClearBuffer();
 
             buffer.Append(leftDel);
             PadLeft(table, line, leftDel, rightDel, columnIndex, buffer);
-            buffer.Append(markdownBold);
-            buffer.Append(line[columnIndex]);
-            buffer.Append(markdownBold);
+            buffer.Append(string.Format(boldMarkupFormat, line[columnIndex]));
```

Comment:
Nit: You could have used `buffer.AppendFormat` instead of `buffer.Append(string.Format(` to simplify it